CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## [0.17.0]
+
+`2021-05-26`
+
+### What's new
+
+- **Added breadcrumbs component.**
+- **Added `.iui-x-small` and `.iui-large` sizes to radial progress indicator.**
+
+## Fixes
+
+- **Fixed wizard responsive design issues.**
+- **Fixed sass deprecation warning about `!global` assignments.**
+
```

Comment:
no need cmoparison addition at the bottom?

Reply:
Added